pkg/util/sys/kmod.go

```diff
@@ -21,6 +21,18 @@ import (
 	"github.com/rook/rook/pkg/util/exec"
 )
 
+const builtinModDir = "/lib/modules/$(uname -r)/modules.builtin"
+
+func IsBuiltinKernelModule(name string, executor exec.Executor) (bool, error) {
+	out, err := executor.ExecuteCommandWithCombinedOutput(false, "", "cat", builtinModDir)
```

Comment:
Can you set the second arg to something meaningful e.g. "kernel modules" or so

Reply:
Thanks for the review. Done!